components/automate-ui/src/app/modules/infra-proxy/cookbooks/cookbooks.component.ts

```diff
@@ -19,10 +19,11 @@ import {
   styleUrls: ['./cookbooks.component.scss']
 })
 
-export class CookbooksComponent implements OnInit {
+export class CookbooksComponent implements OnInit, OnDestroy {
   @Input() serverId: string;
   @Input() orgId: string;
 
+  private isDestroyed = new Subject<boolean>();
```

Comment:
We're still not 100% consistent everywhere yet, but if we can append this with a `$` to visually indicate that it's an observable/subscribable value that'd be great! `isDestroyed$`